packages/dom/src/utils/assert-is-defined.ts

```diff
@@ -0,0 +1,9 @@
+export function assertIsDefined< T >(
```

Comment:
Do we need to add any tests for this one? Could be testing `assertIsDefined` specifically or the code that uses it.

Reply:
Sure!